WordPress/Classes/ViewRelated/Gutenberg/Utils/GutenbergStockPhotos.swift

```diff
@@ -61,7 +61,7 @@ extension GutenbergStockPhotos: StockPhotosPickerDelegate {
         }
 
         let mediaInfo = assets.compactMap({ (asset) -> MediaInfo? in
-            guard let media = self.mediaInserter.insert(exportableAsset: asset, source: .giphy) else {
+            guard let media = self.mediaInserter.insert(exportableAsset: asset, source: .stockPhotos) else {
```

Comment:
Since both instances of this `source` are fixed separately in https://github.com/wordpress-mobile/WordPress-iOS/pull/13940, the changes should be removed from this PR.

Reply:
Thanks for noticing this @ScoutHarris. I reverted those changes.